test/statistics.jl

```diff
@@ -96,6 +100,9 @@ end
 @test_approx_eq var([1], 1; mean=[2], corrected=false) [1.0]
 
 @test var(1:8) == 6.
+@test varm(1:8,1) == var(1:8)
```

Comment:
oops. This is what happens when you write tests to procrastinate. Should we call `var(collect(v),m)` instead?